ssl/ssl_local.h

```diff
@@ -556,6 +556,21 @@ struct ssl_session_st {
     uint32_t flags;
     SSL_CTX *owner;
 
+# ifndef OPENSSL_NO_QUIC
+    /* Remote QUIC transport parameters for 0-RTT, for client only */
+    struct {
+        uint64_t init_max_data;
+        uint64_t init_max_stream_data_bidi_local;
+        uint64_t init_max_stream_data_bidi_remote;
+        uint64_t init_max_stream_data_uni;
+        uint64_t max_local_streams_bidi;
+        uint64_t max_local_streams_uni;
+        uint64_t max_idle_timeout;
+        uint64_t max_udp_payload_size;
+        uint64_t active_conn_id_limit;
+    } quic_params;
```

Comment:
Don't you also need to update the ASN1_SEQUENCE(SSL_SESSION_ASN1) struct below this so that the asn1 encoder knows how to serialize this data?  Or is that happening in a subsequent PR?

Doing that might also necessitate breaking these fields out of the struct you've defined for them, or definig an ASN1 sequence for them, so the parse tree knows how to handle the struct